Source/WebCore/platform/graphics/GraphicsContext.h

```diff
@@ -86,6 +86,12 @@ class GraphicsContext {
 
     virtual bool hasPlatformContext() const { return false; }
     virtual PlatformGraphicsContext* platformContext() const { return nullptr; }
+#if USE(CG)
+    RetainPtr<PlatformGraphicsContext> protectedPlatformContext() const { return platformContext(); }
+#else
+    // On other platforms, the PlatformGraphicsContext type is not refcounted.
+    PlatformGraphicsContext* protectedPlatformContext() const { return platformContext(); }
+#endif
```

Comment:
Now that this is platform-specific I'm not sure it needs an abstract name. Might be easier to understand with a concrete one:
```suggestion
#if USE(CG)
    RetainPtr<CGContextRef> protectedCGContext() const { return platformContext(); }
#endif
```